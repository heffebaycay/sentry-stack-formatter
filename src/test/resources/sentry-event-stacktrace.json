{
  "id": "abad1dea",
  "project": 1337,
  "release": null,
  "platform": "java",
  "culprit": "fr.heffebaycay.nice.package.here.MyCulprit.method(MyCulprit.java:25)",
  "message": "Something bad happened in your application",
  "datetime": "2015-12-02T20:00:02.000000Z",
  "time_spent": null,
  "tags": [
    [
      "tagName",
      "value"
    ],
    [
      "secondTag",
      "secondValue"
    ]
  ],
  "errors": [],
  "extra": {},
  "fingerprint": "{{ default }}",
  "sentry.interfaces.Message": {
    "message": "Something bad happened in your application",
    "params": [
    ]
  },
  "sentry.interfaces.Stacktrace": {
    "frames": [
      {
        "function": "process",
        "lineno": 190,
        "module": "fr.heffebaycay.logic.RequestProcessor",
        "in_app": false
      },
      {
        "function": "doStuff",
        "lineno": 19,
        "module": "fr.heffebaycay.bogus.AwesomeClass",
        "in_app": false
      },
      {
        "function": "callCulprit",
        "lineno": 150,
        "module": "fr.heffebaycay.nice.package.CulpritCaller",
        "in_app": false
      },
      {
        "function": "method",
        "lineno": 25,
        "module": "fr.heffebaycay.nice.package.here.MyCulprit",
        "in_app": false
      }
    ],
    "has_system_frames": false,
    "frames_omitted": null
  },
  "version": "4"
}