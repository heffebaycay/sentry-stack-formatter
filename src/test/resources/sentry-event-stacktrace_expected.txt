Something bad happened in your application
	at fr.heffebaycay.nice.package.here.MyCulprit.method(:25)
	at fr.heffebaycay.nice.package.CulpritCaller.callCulprit(:150)
	at fr.heffebaycay.bogus.AwesomeClass.doStuff(:19)
	at fr.heffebaycay.logic.RequestProcessor.process(:190)
